include/pybind11/eigen.h

```diff
@@ -50,8 +50,12 @@ PYBIND11_NAMESPACE_BEGIN(detail)
 
 #if EIGEN_VERSION_AT_LEAST(3,3,0)
 using EigenIndex = Eigen::Index;
+template<typename Scalar, int Flags, typename StorageIndex>
+using eigen_mapped_sparse_matrix = Eigen::Map<Eigen::SparseMatrix<Scalar, Flags, StorageIndex>>;
```

Comment:
For consistency with the line above, how about EigenMapSparseMatrix?

Reply:
I was aiming for consistency with the numerous other template alias below line 61.

Reply:
I don't think this is public, so no strong opinion. But those are combining snake_case stdlib names and Eigen, while this is purely an Eigen CamelCase name.